cranelift/codegen/src/ir/jumptable.rs

```diff
@@ -23,42 +27,59 @@ pub struct JumpTableData {
 
 impl JumpTableData {
     /// Create a new empty jump table.
-    pub fn new() -> Self {
-        Self { table: Vec::new() }
+    pub fn new(def: Block) -> Self {
```

Comment:
This sounds like define or definition to me. Default is a keyword, and default_ isn't the nicest either. Unless someone has a better suggestion I guess leaving it the way it is right now is fine.